src/sass/_mixins.scss

```diff
@@ -119,3 +119,11 @@
 @function gutter($size){
   @return $size * $layoutDefaultPadding;
 }
+
+@function strip-units($number) {
+  @return $number / ($number * 0 + 1);
```

Comment:
https://css-tricks.com/snippets/sass/strip-unit-function/ - don't you need these ifs there?